packages/formula-editor/src/formula-editor.ts

```
@@ -0,0 +1,205 @@
+
+  /**
+   * user input event type
+   */
+  @state()
+  lastInputType: string;
+
+  @state()
+  _selectedRecommendation: string;
+
+  @state()
+  isFocused: Boolean = false;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)